proto/provenance/msgfees/v1/query.proto

```diff
@@ -0,0 +1,81 @@
+syntax = "proto3";
+package provenance.msgfees.v1;
+
+import "gogoproto/gogo.proto";
+import "google/protobuf/any.proto";
+import "google/api/annotations.proto";
+import "provenance/msgfees/v1/msgfees.proto";
+import "cosmos_proto/cosmos.proto";
+import "cosmos/base/v1beta1/coin.proto";
+import "cosmos/base/query/v1beta1/pagination.proto";
+
+option go_package          = "github.com/provenance-io/provenance/x/msgfees/types";
+option java_package        = "io.provenance.msgfees.v1";
+option java_multiple_files = true;
+
+// Query defines the gRPC querier service for marker module.
+service Query {
+  // Params queries the parameters for x/msgfees
+  rpc Params(QueryParamsRequest) returns (QueryParamsResponse) {
+    option (google.api.http).get = "/provenance/msgfees/v1/params";
+  }
+
+  // Query all Msgs which have fees associated with them.
+  rpc QueryAllMsgFees(QueryAllMsgFeesRequest) returns (QueryAllMsgFeesResponse) {
+    option (google.api.http).get = "/provenance/msgfees/v1/params";
```

Comment:
This needs to be something different than the Params endpoint.
```suggestion
    option (google.api.http).get = "/provenance/msgfees/v1/all";
```

Reply:
good catch probably breaks reverse proxies